### PR TITLE
fix: django import error

### PR DESCRIPTION
**Pull request by src-r-r**
## Description

Just a simple PR to fix the import error. Line 5-9 are the "real" changes.

All the other changes you see are just the linter cleaning up the code.

## Related resources

* https://github.com/django-cms/djangocms-bootstrap4/issues/161

## Checklist

* [x] I have opened this pull request against ``master``
* [x] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined #workgroup-pr-review on 
[Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

**Review by fsbraun — CHANGES_REQUESTED** [review]
@src-r-r Thank you for the PR. 
Can I suggest replacing `ungettext` by `ngettext` everywhere as opposed to importing as? The unicode functions are gone for good.

**Comment by src-r-r** [comment]
Yikes, sorry about that, been stressing out about finding sustainable work so haven't gotten to it. It's super-quiet today so I'll work on it (BTW, if you happen to know of anyone who needs a team lead, feel free to reach out).

File: djangocms_bootstrap4/contrib/bootstrap4_grid/models.py [hunk 1]
```diff
@@ -2,17 +2,26 @@
 
 from django.db import models
 from django.utils.translation import gettext_lazy as _
-from django.utils.translation import ungettext
+
+from django.utils.translation import ngettext as ngettext
```
Comment (fsbraun) [inline djangocms_bootstrap4/contrib/bootstrap4_grid/models.py]:
```suggestion
from django.utils.translation import ngettext
```

**Review by fsbraun — CHANGES_REQUESTED** [review]
Thank you for the update! 

I guess the `ngettext as ngettext` is probably unnecessary.

Also, you'll have to remove `mptt` from the test settings, since `mptt` is gone from django-filer. (See here: https://github.com/django-cms/djangocms-bootstrap4/blob/2c095d31a51ed05c76e2ef528c7ed7c25f6ae94c/tests/settings.py#L7)

**Review by fsbraun — APPROVED** [review]

**Comment by brenosss** [comment]
HI, it looks like this change is required to use Django 4.2. Is there a plan to release those changes?

**Pull request merged**